src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
+LATIN_PUNCT = re.compile(r"[\\/':;,!\?\._『@#\$%^&\*]+")
+QUOTES = re.compile(r'["”“]+')
+# we will extact content between brackets, it is editorial
+ANGLE_BRACKETS = re.compile(r"([a-zA-Z]+)?<[a-zA-Z\s]+>([,\?\.a-zA-Z]+)?")
+SQUARE_BRACKETS = re.compile(r"\[[^\]]+\]")
+OBELIZED_WORDS = re.compile(r"†[^†]+†")
+OBELIZED_WORD = re.compile(r"†[^\s]+\s")
+OBELIZED_PLUS_WORDS = re.compile(r"[\+][^\+]+[\+]")
+OBELIZED_PLUS_WORD = re.compile(r"[\+][^\s]+\s")
+HYPHENATED = re.compile(r"\s[^-]+-[^-]+\s")
+
+
+class JVReplacer:  # pylint: disable=too-few-public-methods
+    """Replace J/V with I/U.
+    Latin alphabet does not distinguish between J/j and I/i and V/v and U/u;
+    Yet, many texts bear the influence of later editors and the predilections of other languages.
+
+    In practical terms, the JV substitution is recommended on all Latin text preprocessing; it
+    helps to collapse the search space.
+
+    >>> replacer = JVReplacer()
+    >>> replacer.replace("Julius Caesar")
+    'Iulius Caesar'
+
+    >>> replacer.replace("In vino veritas.")
+    'In uino ueritas.'
+
+    """
+
+    def __init__(self):
+        """Initialization for JVReplacer, reads replacement pattern tuple."""
+        patterns = [(r"j", "i"), (r"v", "u"), (r"J", "I"), (r"V", "U")]
+        self.patterns = [(re.compile(regex), repl) for (regex, repl) in patterns]
+
+    def replace(self, text):
+        """Do j/v replacement"""
+        for (pattern, repl) in self.patterns:
+            text = re.subn(pattern, repl, text)[0]
+        return text
+
+
+jvreplacer = JVReplacer()
+
+
+class AEOEReplacer:  # pylint: disable=too-few-public-methods
+    """Replace 'œæ' with AE, 'Œ Æ' with OE.
+    Classical Latin wrote the o and e separately (as has today again become the general practice),
+    but the ligature was used by medieval and early modern writings, in part because the
+    diphthongal sound had, by Late Latin, merged into the sound [e].
+    See: https://en.wikipedia.org/wiki/%C5%92
+    Æ (minuscule: æ) is a grapheme named æsc or ash, formed from the letters a and e, originally
+    a ligature representing the Latin diphthong ae. It has been promoted to the full status of a
+    letter in the alphabets of some languages, including Danish, Norwegian, Icelandic, and Faroese.
+    See: https://en.wikipedia.org/wiki/%C3%86
+
+    >>> replacer = AEOEReplacer()
+    >>> replacer.replace("mæd")
+    'maed'
+
+    >>> replacer.replace("prœil")
+    'proeil'
+
+    """
+
+    def __init__(self):
+        """Initialization for AEOEReplacer, reads replacement pattern tuple."""
+        patterns = [(r"œ", "oe"), (r"æ", "ae"), (r"Œ", "OE"), (r"Æ", "AE")]
+        self.patterns = [(re.compile(regex), repl) for (regex, repl) in patterns]
+
+    def replace(self, text):
+        """Do character replacement."""
+        for (pattern, repl) in self.patterns:
+            text = re.subn(pattern, repl, text)[0]
+        return text
+
+
+aeoereplacer = AEOEReplacer()
```

Comment:
This should be caps, but could we rename this to `AEOEREPLACER`. 

https://en.wikipedia.org/wiki/%C5%92#Latin

Reply:
Concur with the renaming.
Normally I agree, no module level default class instantiations--but, here I claim it just wraps an re.compile expression which we often do at module level scope. The problem with putting it into a function is that then it gets called in a loop and leads to poor performance.